### DISCUSSION TITLE
Hey I am new to Github? How to use this

### DISCUSSION THREAD

**Discussion by SiddharthhD** [HarshitKumar9030/Tailwind-Components#8] [Q&A] [answered]
I just started using it, I build projects with arduino and nodejs , Could you tell me how to use  it

**Comment by HarshitKumar9030** [comment]
That was fast

**Comment by HarshitKumar9030** [comment]
Ok, So you use arduino cool

I think for the first place you came here to get some code (isn't it), this isn't all about github

**Comment by SiddharthhD** [comment]
Yeah I know

**Comment by HarshitKumar9030** [comment]
You can start by learning basic things in GIT then maybe you can move to some advanced features and then in github you can explore its api and add things like issues and discussions in your repos also and also can build apps with it.

**Comment by HarshitKumar9030** [comment, marked as answer]
Thanks
